### ISSUE TITLE
Remove warning from PSF docs page

### ISSUE THREAD

**Issue by bsipocz** [astropy/photutils#985]
Probably the API is stable enough by now to remove the red warning note from the top of the docs page?

https://photutils.readthedocs.io/en/stable/psf.html

**Comment by Onoddil** [comment]
As I understood it the API was to be finalised and/or stabilised with the documentation in PRs #721, #753, #755, #756, and #766, which currently require review (#721 having had one pass but further updates still subject to secondary review; we also don't quite have a roadmap for `SceneMaker` which should probably still have a warning note somewhere once it starts to exist). These were supposed to be the foundation of the PSF API on to which all future developments were mapped, I think.

However, if anyone feels that's not the case (@larrybradley @eteq presumably?) of course feel free to remove the label before we finish the documentation, but that's what I took from @eteq's direction when I started on that work previously.
